scripts/globals/mobskills/citadel_buster.lua

```diff
@@ -0,0 +1,89 @@
+---------------------------------------------
+-- Citadel Buster
+-- Deals extreme Light damage to players in an area of effect.
+-- Additional effect: Enmity reset
+-- Damage can be approximated based on Calculating Weapon Skill Damage as a magical WS with a level of 85, fTP of 6 and MAB of 4.0. Or, more simply:
+-- 2088/(1+MDB%) * (256-MDT)/256 (no day/weather bonus)
+-- 2608/MDB * (256-MDT)/256 (weather bonus)
+-- 2816/MDB * (256-MDT)/256 (day+weather bonus)
+---------------------------------------------
+require("scripts/globals/monstertpmoves")
+require("scripts/globals/settings")
+require("scripts/globals/status")
+require("scripts/globals/weather")
+local ID = require("scripts/zones/Temenos/IDs")
+
+---------------------------------------------
+
+function onMobSkillCheck(target,mob,skill)
+    local phase = mob:getLocalVar("battlePhase")
+    if (phase == 4) then
+        mob:setLocalVar("citadelBuster", 1)
+        mob:SetMobAbilityEnabled(false)
+        mob:SetMagicCastingEnabled(false)
+        mob:SetAutoAttackEnabled(false)
+        mob:setMobMod(tpz.mobMod.DRAW_IN, 1)
+        local battlefield = mob:getBattlefield()
+        local players = battlefield:getPlayers()
+        for _, member in pairs(players) do
+            member:messageSpecial(ID.text.CITADEL_BASE)
+        end
+        mob:timer(10000, function(mob)
+            for _, member in pairs(players) do
+                member:messageSpecial(ID.text.CITADEL_BASE+1)
+            end
+            mob:timer(10000, function(mob)
+                for _, member in pairs(players) do
+                    member:messageSpecial(ID.text.CITADEL_BASE+2)
+                end
+                mob:timer(5000, function(mob)
+                    for _, member in pairs(players) do
+                        member:messageSpecial(ID.text.CITADEL_BASE+3)
```

Comment:
Others: I started typing an idea, then spoke with cocosolos on Discord. We might make use of onBattlefieldTick.